vowels-only.js

```diff
@@ -0,0 +1,25 @@
+// Write a function called vowelsOnly which accepts a string str 
+// and returns a new string consisting of just the vowels in str.
+
+function vowelsOnly(str) {
+
+    var matches = str.match(/[aeiou]/gi);
+    // console.log("matches", matches);
+
+    var newStr = "";
```

Comment:
since you are checking if matches exists, you should define this after checking if matches exist. Otherwise, you are creating a variable and never using it.